test/system/projects/samples_test.rb

```diff
@@ -16,7 +16,7 @@ class SamplesTest < ApplicationSystemTestCase
     test 'visiting the index' do
       visit namespace_project_samples_url(namespace_id: @namespace.path, project_id: @project.path)
       assert_selector 'h1', text: I18n.t('projects.samples.index.title')
-      assert_selector 'table#samples-table tr', count: 2
+      assert_selector 'table#samples-table tr', count: 3
```

Comment:
Can the counts remain the same as we did not change the number of samples being displayed? Instead could we change the selector to look inside tbody?

Reply:
All counts have been reversed and selector updated in 767377956c07734fd18058edd4b8dc9ac1af1da1